src/Components/Register/registerValidations.js

```diff
@@ -0,0 +1,72 @@
+// Config
+import config from '../../CritCore/Config/config';
+
+// Translations
+import getTranslations from '../../CritCore/Translations/Translations';
+
+const { language }         = config,
+translations               = getTranslations(language),
+registerInputErrors        = translations.register,
+msgErrorInput              = registerInputErrors.errors,
+msgErrorInputEmail         = msgErrorInput.email,
+msgErrorInputUserName      = msgErrorInput.userName,
+msgErrorInputPassword      = msgErrorInput.password,
+msgErrorInputAlterEgo      = msgErrorInput.alterEgo,
+msgErrorInputIsPwdMatched  = msgErrorInput.isPwdMatched,
+msgErrorInputFieldRequired = msgErrorInput.fieldRequired;
+
+const registerValidator = {
+  email: ({ value }) => {
+    //eslint-disable-next-line
+    const emailRegexp =  /^\w+([\.\+\-]?\w+)*@([\.-]?\w+)*(\.\w{2,4})+$/;
```

Comment:
Same as below

Reply:
I couldn't remove the values ​​by checking them in as variables.
However, we were talking about leaving the minimum / maximum for the email, in order to validate up to 4 characters after the period (.1234)